backend/.env.example

```
@@ -17,5 +17,19 @@ API_BASE_URL=http://localhost:8000
+
+# ── ML Models (optional) ──────────────────────────────────────────────────────
+# Leave unset → demo mode (random scores, no PyTorch needed).
+# Uncomment to enable real ML inference:
+# MODEL_DIR=/absolute/path/to/Models
+# STREAM_A_MODEL=/absolute/path/to/Models/freshscan_stream_a_body.pth
+# STREAM_B_MODEL=/absolute/path/to/Models/stream_b_checkpoint.pth
+
+# ── HF Hub (production HF Space only — set as Space secrets, NOT here) ────────
+# HF_MODEL_REPO=karansingh12/freshscan-models
+# MODEL_TOKEN=hf_xxxxxxxxxxxxxxxxxxxx
+=========
```

dotenv-linter (4.0.0)

[warning] [LeadingCharacter] Invalid leading character detected

(LeadingCharacter)